Bio/motifs/meme.py

```diff
@@ -75,9 +75,12 @@ def __init__(self, alphabet=None, instances=None):
 class Instance(Seq.Seq):
     """A class describing the instances of a MEME motif, and the data thereof."""
 
-    def __init__(self, *args, **kwds):
-        """Initialize the class."""
-        Seq.Seq.__init__(self, *args, **kwds)
+    def __init__(self, sequence, alphabet=None):
```

Comment:
On reflection, this just needs to take the sequence as the sole argument - the calling code does not need to pass any alphabet (even as valid letters).

Reply:
Reworked; updated branch pushed.